packages/python/accern_data/util.py

```diff
@@ -147,6 +149,25 @@ def generate_file_response(
     return response_obj
 
 
+def register_on_record() -> None:
+    directory, folder = os.path.split(os.path.split(__file__)[0])
+    full_directory = os.path.join(
+        directory,
+        f"{folder}-{__version__.replace('-', '')}.dist-info",
+        "RECORD")
+    lines = {
+        f"{folder}/data/data-2022.json",
+        f"{folder}/data/data-2022.csv",
+        f"{folder}/data/.DS_Store",
```

Comment:
those files will only exist on mac -- you have to check if they exist before adding them to the list